Automattic/jetpack
# Update jetpack-boost-critical-css-gen to the latest version

The Jetpack Boost Critical CSS Gen library has been updated to [clean up its main loop](https://github.com/Automattic/jetpack-boost-critical-css-gen/pull/21) and [handle broken CSS files without crashing](https://github.com/Automattic/jetpack-boost-critical-css-gen/pull/22).

This PR updates Boost up to the latest version.

#### Changes proposed in this Pull Request:
* Update Critical CSS gen lib to the latest version to get the latest fixes into Boost.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Make sure Critical CSS generation works.
* Make sure Critical CSS generation doesn't crash with corrupted CSS like this:
```
@font-face {
	src: );
}
```
